cmd/create.go

```diff
@@ -29,12 +29,16 @@ const logo = `
 `
 
 var (
-	logoStyle      = lipgloss.NewStyle().Foreground(lipgloss.Color("#01FAC6")).Bold(true)
-	endingMsgStyle = lipgloss.NewStyle().PaddingLeft(1).Foreground(lipgloss.Color("170")).Bold(true)
+	logoStyle           = lipgloss.NewStyle().Foreground(lipgloss.Color("#01FAC6")).Bold(true)
+	endingMsgStyle      = lipgloss.NewStyle().PaddingLeft(1).Foreground(lipgloss.Color("170")).Bold(true)
+	allowedProjectTypes = []string{"chi", "gin", "fiber", "gorilla/mux", "httprouter", "standard-library", "echo"}
 )
 
 func init() {
 	rootCmd.AddCommand(createCmd)
+
+	createCmd.Flags().StringP("project-name", "n", "", "Name of project to create")
+	createCmd.Flags().StringP("project-type", "t", "", fmt.Sprintf("Type of project to create. Allowed values: %s", strings.Join(allowedProjectTypes, ", ")))
```

Comment:
I have one concern with this - if we want to have flag support we should not allow the CLI to kick off.
It should be one or the other, otherwise we can introduce some nasty BUGS

Reply:
The reasoning behind not splitting up as if you only place the name, we could still show the TUI for selecting the type of project to configure. We could still build it in another way tho, for sure!

Reply:
Hold on I just tested this out and its super nice.